dw-icon-button.js

```diff
@@ -185,18 +198,44 @@ export class DwIconButton extends buttonFocus(LitElement) {
        */
       title: { type: String },
 
+      /**
+       * Input property.
+       * Disabled Tooltip text.
+       */
+      disabledTitle: { type: String },
```

Comment:
It seems you introduced this as a new feature. But, the card's title or scope doesn't mention about it.